procedures/pulpcore/update.md.erb

```diff
@@ -1,14 +1,14 @@
 # Prepare the build environment
 
-* [ ] Clone https://github.com/evgeni/pulpcore-packaging
-* [ ] [Configure](https://github.com/evgeni/pulpcore-packaging#configure) the build environment
-* [ ] [Build](https://github.com/evgeni/pulpcore-packaging#build) the container
+* [ ] Clone https://github.com/theforeman/pulpcore-packaging
+* [ ] [Configure](https://github.com/theforeman/pulpcore-packaging/automation#configure) the build environment
+* [ ] [Build](https://github.com/theforeman/pulpcore-packaging/automation#build) the container
 
-# Create/update packages in `foreman-packaging`
+# Create/update packages in `pulpcore-packaging`
 
-* [ ] [Run](https://github.com/evgeni/pulpcore-packaging#run) the container, passing in the configuration and an empty folder that shall contain the results
+* [ ] [Run](https://github.com/theforeman/pulpcore-packaging/automation#run) the container, passing in the configuration and an empty folder that shall contain the results
 ```
-podman run --rm --volume $(pwd)/foreman-packaging/:/app/foreman-packaging/:Z  --env-file packaging.env tfm-pulpcore-builder
+podman run --rm --volume $(pwd)/packaging/:/app/packaging/:Z  --env-file packaging.env tfm-pulpcore-builder
```

Comment:
I'm assuming $(pwd)/packaging/ is supposed to be the pulp-packaging directory?  This wasn't that clear, and I had to fiddle around a little bit

Reply:
Actually, looks like this just needs to be some empty `packaging` directory. Perhaps an edit at "1. create an empty directory to host the resulting packages" in the automation, or "and an empty folder that shall contain the results" here to better show the alignment of `$(pwd)/packaging/` with this intent

Reply:
it doesn't even have to exist… lemme rephrase that ;)

Reply:
but the line before actually says "an empty folder that shall contain the results" ;)

Reply:
updated, how do you like it now?